hack/kind-cluster.sh

```diff
@@ -34,6 +42,58 @@ nodes:
 - role: worker
 - role: worker
 EOF
+}
+
+# install_operator installs the operator on top of the kind cluster
+install_operator() {
+  pushd ${DIR}/../
+  make install
+  make docker-build IMG=localhost/ingress-node-firewall-controller:local
+  kind load docker-image localhost/ingress-node-firewall-controller:local
+  make docker-build-daemon DAEMON_IMG=localhost/ingress-node-firewall-daemon:local
+  kind load docker-image localhost/ingress-node-firewall-daemon:local
+  set +x
+  hack/set-daemon-image.sh localhost/ingress-node-firewall-daemon:local || then_hit_any_key
```

Comment:
instead of this we can use yq and replace the image directly ?

Reply:
that is using yq under the hood though (plus it installs yq, etc.)

Reply:
The content of that script is:
~~~
[akaris@linux ingress-node-firewall (akarislibxdp)]$ cat hack/set-daemon-image.sh
#!/bin/bash

set -eux

DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
DAEMONSET_IMAGE=$1

if ! command -v yq ; then 
  echo "Could not find yq binary - trying to install it"
  go install github.com/mikefarah/yq/v4@latest
fi

if ! command -v yq ; then 
  echo "Cannot find command yq and could not install it."
  echo "Please modify file config/manager/env.yaml manually and point DAEMONSET_IMAGE to:"
  echo "    ${DAEMONSET_IMAGE}"
fi
yq e "(.spec.template.spec.containers[] | select(.name == \"manager\") .env[] | select(.name == \"DAEMONSET_IMAGE\") ).value=\"${DAEMONSET_IMAGE}\"" \
    -i ${DIR}/../config/manager/env.yaml
~~~

Keeping the script as an individual tool is IMO better than merging it into the kind-cluster.sh script, but that's my personal opinion

Reply:
I have a relatively strong preference for keeping the yq stuff inside set-daemon-image.sh ; especially given that we are keeping the manual steps in our documentation. If your indifferent about it, I'd like to keep it that way. If you have a strong preference for integrating that in kind-cluster.sh, then I'll add the logic to kind-cluster.sh as well (at the cost of having the same logic both in set-daemon-image.sh and kind-cluster.sh). lemme know what you think :-)

Reply:
i was referring to the part where the script pause waiting to enter something